docs/examples/binding_lifetime/binding_lifetime.rst

```diff
@@ -91,6 +91,18 @@ Load the tracks into Pylake::
 
 Use the same approach as above to load the tracks from Lakeview, except that the part :func:`Kymo.crop_by_distance() <lumicks.pylake.kymo.Kymo.crop_by_distance>` has to be removed.
 
+The (loaded) tracks can be plotted on top of the original kymograph to visualize the result of the tracking::
+
+    plt.figure()
+    kymo1_selection.plot("g", aspect = 5, adjustment=lk.ColorAdjustment([0], [5]))
```

Comment:
Minor, but you don't need to make the adjustments lists if you specify only a single value.

```suggestion
    kymo1_selection.plot("g", aspect=5, adjustment=lk.ColorAdjustment(0, 5))
```